### PR TITLE
Styling Of The BookStore

### PR DESCRIPTION
**Pull request by Have-Samuel**
This pull request features the following features

- Most links are able to be hovered over but have no functionality
- Remove link has the functionality, as does Add Book
- Categories were not added as there is no way for a user to input or modify the book category at this time
- Update Progress button is purely aesthetic at this time and has no functionality
- Complete styling of the entire Books SPA page

### CODE REVIEW

File: src/styles/Books.css [hunk 1]
```diff
@@ -0,0 +1,191 @@
...
+  padding: 2rem 9.188rem 1.625rem 1.688rem;
+  border-radius: 4px;
+  border: solid 1px var(--grey-border);
+  background-color: var(--white);
+}
+
+.container {
+  display: flex;
+  justify-content: space-between;
+}
```
Comment (roshan-bajgain) [inline src/styles/Books.css]: - [ ] I would suggest you adjust the spacing from top to button equally. If you see the logo, container, and Add-book then you can find this vertically in the same line.
- [ ] Kindly make the `BookStore CMS` bolder as the design.
### Figma design
figma-design
### Your design
his

File: src/styles/Books.css [hunk 2]
```diff
@@ -0,0 +1,191 @@
...
+  margin: 0.25rem 0 0;
+  font-family: RobotoSlab, sans-serif;
+  font-size: 0.875rem;
+  font-weight: 300;
+  font-stretch: normal;
+  font-style: normal;
+  line-height: normal;
+  letter-spacing: normal;
+  color: var(--cool-blue);
+  cursor: pointer;
```
Comment (roshan-bajgain) [inline src/styles/Books.css]: - [ ] When I add `book` and `author` then It is adding the container horizontally but if you see the design then the container should add one after another like vertically, See the design and Kindly fix it 
### your design
hiss
### Figma design
figma-design

**Review by roshan-bajgain — CHANGES_REQUESTED** [review]
Hi there, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>

### To highlight :tada: :champagne: 
- Nice PR description and title :+1: 
- Nice README file :heavy_check_mark:
- No linter error :heavy_check_mark:
- GitHub flow is followed :+1:


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/components/Books.js [hunk 3]
```diff
@@ -1,21 +1,57 @@
 import PropTypes from 'prop-types';
 import { useDispatch } from 'react-redux';
 import { removeBook } from '../redux/books/books';
+import '../styles/Books.css';
```
Comment (omar-labana) [inline src/components/Books.js:4]: - [ ] The books section is not styled properly. When adding books, the new book is added in a new column which breaks the styling and the page.
image

**Review by omar-labana — CHANGES_REQUESTED** [review]
Hi @Have-Samuel, 

Excellent work so far! 👌
You have implemented almost all the required features. ✔️ 
There are some cases that you need to adjust but you are almost done! 😀

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by OLIPLICHE — APPROVED** [review]
Hi @Have-Samuel , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**